clients/client-java/src/main/java/org/apache/gravitino/client/HTTPClient.java

```diff
@@ -132,18 +132,17 @@ private HTTPClient(
    * @return The response body as a string.
    * @throws RESTException If an error occurs during conversion of the response body to a string.
    */
-  private String extractResponseBodyAsString(CloseableHttpResponse response) {
+  
+private String extractResponseBodyAsString(CloseableHttpResponse response) {
     try {
-      if (response.getEntity() == null) {
-        return null;
-      }
-
-      // EntityUtils.toString returns null when HttpEntity.getContent returns null.
-      return EntityUtils.toString(response.getEntity(), "UTF-8");
+        if (response.getEntity() == null) {
+            return null;
+        }
+        return EntityUtils.toString(response.getEntity(), StandardCharsets.UTF_8);
     } catch (IOException | ParseException e) {
-      throw new RESTException(e, "Failed to convert HTTP response body to string");
```

Comment:
This is a format change, I guess it will break the spotless check rules, did you ever run gradle successfully after making this change?